kops

```diff
@@ -273,13 +284,19 @@ def _cmd_create(args):
     cluster_numbers = sorted(cluster_numbers)
     for cluster_number in range(1, 100):
         if cluster_number not in cluster_numbers:
-            break
+            return cluster_number
     else:
         _error('Couldn\'t determine a unique cluster name')
 
+def _cmd_create(args):
+    """Invoke kops to create cluster."""
+    cluster_name = '{}.{}'.format(args.category, args.name)
+
+    cluster_number =_check_existing_cluster(args, cluster_name)
+
     # Use three availability zones and one master for each in order to obtain
     # high availability
-    zones = 'eu-central-1a,eu-central-1b,eu-central-1c'
+    zones = 'us-west-2a,us-west-2b,us-west-2c'
```

Comment:
Why do you change these zones? From reviewing the code, I think the zones should be configurable by the user though instead of hardwired. I would make these values parameterized in another PR.

Reply:
I agree, the zones and machine size should be configurable. I first wanted to be sure that my PR was on the desired direction before making more changes.